### DISCUSSION TITLE
Welcome to ExpandedHelicopterEvents Discussions!

### DISCUSSION THREAD

**Discussion by Chuckleberry-Finn** [TEHE-Studios/ExpandedHelicopterEvents#98] [Announcements]

**Comment by ASHBLY33** [comment]
How can I know wich helicopter will be arriving in base to what AEBS says?

**Reply by Chuckleberry-Finn** [reply]
Each event has a particular message. Should be pretty straight forward, but you can root around the mod files to see. [Radio translations](https://github.com/TEHE-Studios/ExpandedHelicopterEvents/blob/main/Contents/mods/Expanded%20Helicopter%20Events/media/lua/shared/Translate/EN/DynamicRadio_EN.txt)

**Comment by woodyafortunato** [comment]
Hello on my mp server the siren keeps blaring and won't stop

**Reply by Chuckleberry-Finn** [reply]
Make sure the mod is updated. That issue should be fixed.

**Comment by Kangarumpy** [comment]
Been hosting a server for a few friends when the update dropped, but...
The latest update broke Steam/Workshop:
Resubbing does not add the . . \steamapps\common\ProjectZomboid\steamapps\workshop\content\108600\2458631365 folder, and manually installing said folder via a friend sending me their copy of the folder causes the folder to get deleted and Zomboid saying "Server has stopped during launch (Normal Termination)"

Server log attached:
[20-02-23_00-28-26_DebugLog-server.txt](https://github.com/TEHE-Studios/ExpandedHelicopterEvents/files/10779911/20-02-23_00-28-26_DebugLog-server.txt)

**Reply by IIAkariII** [reply]
The same thing happened to me, I have only been able to solve it by uninstalling the entire game, deleting the entire folder, and then reinstalling everything :/

**Reply by Kangarumpy** [reply]
Thank you for *a* solution. I should get things set back up by tonight for my friends because of you!

**Comment by CharliiSteam** [comment]
It would be nice to see random crashes, not by weather condition. for example: pilot/co-pilot zombified because of the airborn infection

**Comment by doctorthoss** [comment]
As server admin is it possible to shorten the duration of or force end a helicopter event? the supply drop event in particular seems to last a really long time if you don't go and get the drop (which I usually can't do because the HUD icon disappears too quick), and doesn't even seem to draw many zombies anyway

**Reply by Chuckleberry-Finn** [reply]
Sounds like the event sound is getting stuck.

**Comment by DewclawTV** [comment]
Love this mod.  Adds a ton of life to the world.  I enjoy playing with infinite rain, so I think that causes the weather to be 'too poor' for the events to happen as often as I'd like even with settings set as high as I can make them for military and police.  Is there a chance we could get the option to toggle and set a % chance of crashes for poor weather?

**Reply by Chuckleberry-Finn** [reply]
This was suggested before and I forgot to add it on the to-do.

**Comment by GhostLord21** [comment]
Was wondering if you can ride the helicopters, I've never seen one in a parking lot or in a mili base so was wondering if they actually spawn?

**Comment by sharkster91** [comment]
No, you cannot.
